cats-effect/src/main/scala/neotypes/cats/implicits.scala

```diff
@@ -1,7 +1,9 @@
 package neotypes.cats
 
-object implicits {
-  implicit def catsAsync[F[_]](implicit F: cats.effect.Async[F]): neotypes.Async[F] =
+object implicits extends CatsInstances
+
+private[cats] abstract class CatsInstances {
```

Comment:
Yes I was going to do this in the next PR.
See the end of this [comment](https://github.com/neotypes/neotypes/issues/35#issuecomment-506402695)

> Also, I just noticed that, the implicits of each module should also be splitted in both a trait and an object extending such trait.